src/cpp/rtps/builtin/discovery/endpoint/EDPSimple.cpp

```diff
@@ -56,9 +56,9 @@ namespace rtps {
 
 // Default configuration values for EDP entities.
 static const dds::Duration_t edp_heartbeat_period{1, 0}; // 1 second
-static const dds::Duration_t edp_nack_response_delay{0, 100 * 1000 }; // 100 milliseconds
-static const dds::Duration_t edp_nack_supression_duration{0, 10 * 1000}; // 10 milliseconds
-static const dds::Duration_t edp_heartbeat_response_delay{0, 10 * 1000}; // 10 milliseconds
+static const dds::Duration_t edp_nack_response_delay{0, 100 * 1000000 }; // 100 milliseconds
+static const dds::Duration_t edp_nack_supression_duration{0, 10 * 1000000}; // 10 milliseconds
+static const dds::Duration_t edp_heartbeat_response_delay{0, 10 * 1000000}; // 10 milliseconds
```

Comment:
The values changed in commit 39fa6011d84e311060cb698d5b4e0ac4efc96163. The comment was updated, but the value was wrongly converted.

I prefer the old format (multiplying twice by 1000):

```suggestion
static const dds::Duration_t edp_nack_response_delay{0, 100 * 1000 * 1000 }; // 100 milliseconds
static const dds::Duration_t edp_nack_supression_duration{0, 10 * 1000 * 1000}; // 10 milliseconds
static const dds::Duration_t edp_heartbeat_response_delay{0, 10 * 1000 * 1000}; // 10 milliseconds
```